Directory.Build.props

```diff
@@ -45,13 +45,15 @@
     <OutputPath>$(BaseOutputPath)\$(Configuration)</OutputPath>
     
     <!-- Path for tools output files, .exe executable generated files for tools can be found here  -->
-    <BuildToolsPath>$(BaseOutputPath)\tools\$(NetfxVersion)</BuildToolsPath>
+    <BuildToolsPath>$(BaseOutputPath)\tools\$(NetCoreVersion)</BuildToolsPath>
```

Comment:
I had moved the build tools to only netcore since they're not included in the nupkg.  Do we want them in both netcore and net framework because we have a build environment with only net framework? I may very well be missing something.